### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: streams/us.m3u [hunk 1]
```diff
@@ -210,10 +210,12 @@ https://vblive-c.viebit.com/5f0d9ca5-4e85-4c01-a426-9ec8d44c2c9c/chunklist.m3u8
 https://d2up1hmow19bcd.cloudfront.net/livecf/liveracing/playlist.m3u8
 #EXTINF:-1 tvg-id="CartoonNetworkAsia.sg" status="timeout",Cartoon Network Asia (720p)
 http://198.16.106.62:8278/streams/d/Cn/playlist.m3u8
-#EXTINF:-1 tvg-id="CartoonNetworkEast.us" status="online",Cartoon Network East TVE Feed (720p) [Geo-Blocked] [Not 24/7]
+#EXTINF:-1 tvg-id="CartoonNetworkEast.us" status="blocked",Cartoon Network East TVE Feed (720p) [Geo-Blocked] [Not 24/7]
 https://tve-live-lln.warnermediacdn.com/hls/live/2023167/tooneast/slate/VIDEO_0_3564000.m3u8
-#EXTINF:-1 tvg-id="CartoonNetworkEast.us" status="online",Cartoon Network East TVE Feed (480p) [Geo-Blocked] [Not 24/7]
+#EXTINF:-1 tvg-id="CartoonNetworkEast.us" status="blocked",Cartoon Network East TVE Feed (480p) [Geo-Blocked] [Not 24/7]
 https://tve-live-lln.warnermediacdn.com/hls/live/2023167/tooneast/slate/VIDEO_2_1964000.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkEast.us" status="online",Cartoon Network East (720p)
```
Comment (LaneSh4d0w) [inline streams/us.m3u:217]:
```suggestion
#EXTINF:-1 tvg-id="CartoonNetworkEast.us" status="online",Cartoon Network East (720p)
#EXTVLCOPT:http-referrer=https://tv247.us/
```

**Review by ghost — COMMENTED** [review]
bb

**Comment by bbbureneecartoonz** [comment]
can you do merging and reviewing pls?

**Review by ghost — COMMENTED** [review]
sfdgr

**Review by ghost — COMMENTED** [review]
ggfhfgh

**Review by BellezaEmporium — APPROVED** [review]
I suppose it's a soft geo-lock for this one, but link seems to work with the corresponding referer. (Goes through a fake Uplynk site trying to load CN)

**Review by what-a — COMMENTED** [review]
The two streams which are proposed to change from "online" to "blocked" play fine for me (in the USA). I'm thinking these should remain as "online."

**Comment by LaneSh4d0w** [comment]
> The two streams which are proposed to change from "online" to "blocked" play fine for me (in the USA). I'm thinking these should remain as "online."

The CN programs only work for people inside the USA.

**Review by what-a — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
bb

**Comment by bbbureneecartoonz** [comment]
merge pls

**Pull request merged**